plugins/github.js

```diff
@@ -10,24 +10,35 @@ exports.init = function(config, cimpler) {
          return next();
       }
 
-      try {
-         var build = extractBuildInfo(req.body);
-         if (build) {
+      const payload = JSON.parse(req.body.payload);
+      const event = req.headers['X-GitHub-Event'];
+      var build = null;
+
+      /**
+       * The webhook sent must be a push event OR a pull_request event,
+       * not both.
+       */
+      if (event == 'push') {
```

Comment:
does a `push` only happen on pull requests?

Reply:
it never happens on pull_request

Reply:
2 different events

Reply:
look at the headers: 
https://github.com/iFixit/ifixit/settings/hooks/199891646
https://github.com/iFixit/ifixit/settings/hooks/311305